parsing/pcoq.ml

```diff
@@ -587,12 +587,31 @@ let recover_grammar_command (type a) (tag : a grammar_command) : a list =
 
 let extend_dyn_grammar (GrammarCommand.Dyn (tag, g)) = extend_grammar_command tag g
 
+(* Tactic parsing modes *)
+let add_proof_tactic_entry p e = Hashtbl.add tactic_entry p e
```

Comment:
Indeed, the API is still very preliminar so I expect it'll have to greatly evolve to be able to put GEXTEND on their own table.

A harder task will be to switch parser state using a rule...